handler/handler_test.go

```diff
@@ -338,7 +338,7 @@ func TestClient_Nearest(t *testing.T) {
 			if tt.cl == nil {
 				tt.cl = clientgeo.NewAppEngineLocator()
 			}
-			c := NewClient(tt.project, tt.signer, tt.locator, tt.cl, prom.NewAPI(nil), tt.limits, tt.ipLimiter, nil)
+			c := NewClient(tt.project, tt.signer, tt.locator, tt.cl, prom.NewAPI(nil), tt.limits, tt.ipLimiter, nil, nil)
```

Comment:
🤔 I am not super happy about clearing the path for passing `nil` to types that should not be `nil` even for testing. Perhaps, you can just instantiate the simplest possible verifier?